ajax/libs/amcharts/package.json

```diff
@@ -2,13 +2,34 @@
   "name": "amcharts",
   "filename": "amcharts.js",
   "version": "3.13.0",
-  "description": "amCharts is a robust charting tool that will suit any dataviz need.",
+  "description": "JavaScript Charts V3",
   "keywords": [
-    "chart"
+    "amcharts",
+    "javascript",
+    "charts"
   ],
-  "homepage": "http://www.amcharts.com/",
+  "homepage": "https://www.amcharts.com/",
+  "author": "amCharts <contact@amcharts.com>",
   "license": {
     "name": "Free and commercial",
-    "url": "http://www.amcharts.com/download/"
+    "url": "https://www.amcharts.com/download/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/amcharts/amcharts3.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/amcharts/amcharts3.git",
+    "fileMap": [
+      {
+        "basePath": "amcharts",
+        "files": [
+          "patterns/!(chalk)/**",
```

Comment:
@maruilian11 I think we can just use single star like `"patterns/!(chalk)/*",`. thank you.